### ISSUE TITLE
Can you do it locally in Python?

### ISSUE THREAD

**Issue by stphtan94117** [bao18/open_earth_map#1]
Colab's network is not friendly to me, so can you make python(.py) on computer?
Can you please add the code how to output the result?
thanks

**Comment by bao18** [comment]
Thanks for your interest in the OpenEarthMap project. I have added sample python codes for training and testing.

**Comment by stphtan94117** [comment]
i download OpenEarthMap_Mini.zip and try to train.
it is error on training.
you can see the screenshots. 
![image](https://user-images.githubusercontent.com/110151053/207273179-75a389ed-b92c-42eb-a7aa-9f2cf6988bf4.png)
![image](https://user-images.githubusercontent.com/110151053/207273248-d8e7ec45-aff1-4214-b9e3-6d922e28cc7b.png)
![image](https://user-images.githubusercontent.com/110151053/207273428-5fa6841b-2d63-4bd2-bb5d-fc49feaceb8e.png)
![image](https://user-images.githubusercontent.com/110151053/207273469-1dca4c9f-8d90-407a-b30d-00aecea35ac4.png)

**Comment by bao18** [comment]
Make sure that the OEM_DATA_DIR is correctly set.

**Comment by stphtan94117** [comment]
> Make sure that the OEM_DATA_DIR is correctly set.

i find the reason why training is error, line 28 need to remove"/"
when i remove / ,it will finish training.
![image](https://user-images.githubusercontent.com/110151053/207482177-63626cd3-e820-4490-8ec0-205896a1f0f0.png)
but when i test, it still error.
i did not know how to solve.

![image](https://user-images.githubusercontent.com/110151053/207482347-e7c18b68-cc60-4671-9999-13df506fa744.png)

**Comment by bao18** [comment]
Yeah, it is the same type of error as the previous one. Please make sure that the "`fout`" in the testing demo code/notebook is correctly set.
